components/create-collective/sections/CreateCollectiveForm.js

```diff
@@ -0,0 +1,227 @@
+                      label="What's the name of your collective?"
+                      value={values.name}
+                      required
+                      m={4}
+                      p={2}
+                    >
+                      {inputProps => <Field as={StyledInput} {...inputProps} placeholder="Guinea Pigs United" />}
+                    </StyledInputField>
+                    <StyledInputField
+                      name="url"
+                      htmlFor="url"
+                      error={errors.url}
+                      label="What URL would you like?"
+                      value={values.url}
+                      required
+                      m={4}
+                      p={2}
+                    >
+                      {inputProps => (
+                        <Field
+                          as={StyledInputGroup}
+                          {...inputProps}
+                          prepend="opencollective.com"
+                          placeholder="guineapigs"
+                        />
+                      )}
+                    </StyledInputField>
+                    <StyledInputField
+                      name="desc"
+                      htmlFor="desc"
+                      error={errors.desc}
+                      label="What does your collective do?"
+                      value={values.desc}
+                      required
+                      m={4}
+                      p={2}
+                    >
+                      {inputProps => (
+                        <Field
+                          as={StyledInput}
+                          {...inputProps}
+                          placeholder="We advocate for tiny piggies everywhere!"
+                        />
+                      )}
+                    </StyledInputField>
+
+                    <Box className="tos" m={4} p={2}>
+                      <label>{intl.formatMessage(this.messages['tos.label'])}</label>
+                      <div>
+                        <input
+                          type="checkbox"
+                          name="tos"
+                          required
+                          onChange={({ target }) => this.handleChange('tos', target.checked)}
+                        />
+                        <span>
+                          I agree with the{' '}
+                          <a href="/tos" target="_blank" rel="noopener noreferrer">
+                            terms of service of Open Collective
+                          </a>
+                        </span>
+                      </div>
+                      {get(host, 'settings.tos') && (
+                        <div>
+                          <StyledCheckbox required onChange={({ checked }) => this.setState({ checked })} />
+                          <span>
+                            I agree with the{' '}
+                            <a href={get(host, 'settings.tos')} target="_blank" rel="noopener noreferrer">
```

Comment:
This will need to be localized.

Reply:
Also, the "I agree with the ..." should be clickable and activate the checkbox.Seems it needs to be passed as `label`?

Reply:
Yes to localising - haven't done all of them as I go which probably isn't best practice, sorry 🙈 

I did some work on another checkbox today and figured out best practice so will update this one too.